resolwe_bio/processes/variant_calling/create_genomicsdb.py

```diff
@@ -169,6 +182,17 @@ def run(self, inputs, outputs):
         else:
             self.error("Intervals file is required for creating a new database.")
 
+        temp_dir = Path("temp/")
```

Comment:
Better approach is to use the temp dir as defined by env variables: `temp_dir = os.environ.get("TMPDIR")`

Reply:
Thanks, I will replace it.